.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [16.x, 15.x, 14.x, 13.x, 12.x]
+        node: [16.x, 14.x, 12.x, "12.22.0"]
         include:
```

Comment:
We can also remove this:

https://github.com/eslint/espree/blob/b068cea907ef59fe5681f6acedd30b932d496e0a/.github/workflows/ci.yml#L48-L65